CONTRIBUTING.md

```diff
@@ -46,6 +46,45 @@ Make sure you have the following dependencies installed first:
    mage -v
    ```
 
+## Local development with Grafana
+
+For developing or debugging a plugin, it's useful to link it with a local copy of Grafana so you can see the changes while you are adding them. To do so:
+
+1. Clone [Grafana](https://github.com/grafana/grafana)
+
+   a. [optional] Checkout to the specific version you want to target (e.g. `git checkout v8.3.4`)
+
+2. Install its dependencies
+
+   ```bash
+   yarn install
+   ```
+
+3. Grafana will look for plugins, by default, on its `data/plugins` directory. You can create a symbolic link to your plugin repository to detect new changes:
```

Comment:
Is it worth mentioning that it can be done in a `conf/custom.ini` file too?
Or would you say a symlink is a preferred way?

```
[paths]
provisioning = /Users/xxx/code/plugin-provisioning/provisioning
 
plugins = /Users/xxx/code/plugins
```

Reply:
both approaches are valid :+1: